README.md

```diff
@@ -2,25 +2,49 @@
 [![Build Status](https://travis-ci.org/korteke/Shibboleth-IdP3-TOTP-Auth.svg?branch=master)](https://travis-ci.org/korteke/Shibboleth-IdP3-TOTP-Auth)
 
 # Shibboleth-IdP3-TOTP-Auth
-> Working example of the TOTP authenticator. Work in progress! Refactoring needed! Localization needed.  
+> Working example of the TOTP authenticator. Work in progress! Refactoring
+> needed! Localization needed.
 
-Google authenticator authentication module for Shibboleth IdP v3.  
-Works conjunction with the User/Password flow. This module first calls authn/Password flow and after that flow is completed it asks token code from the user. User can also register a new token with this module.  
+Google authenticator authentication module for Shibboleth IdP v3.
+Works conjunction with the User/Password flow. This module first calls
+authn/Password flow and after that flow is completed it asks token code from the
+user. User can also register a new token with this module.
 
-Uses External LDAP, MongoDB(EXPERIMENTAL!) or Static for seed fetching.  
+Uses External LDAP, MongoDB(EXPERIMENTAL!) or Static for seed fetching.
 
 Working example with Vagrant https://github.com/korteke/shibboleth-vagrant
 
 Requirements
 ------------
 
-Shibboleth IdP v3.2.x  
+Shibboleth IdP v3.2.x
 Java 8
 
 Installing
 ----------
 
-* Compile souce code with maven - ```mvn clean package```
```

Comment:
Original this command was providing a `*.zip` file within a targets directory. It doesn't seem like the `gradle init` really did a good job of migrating. but I'm also not sure if I will need a plugin or something to generate that ZIP file.

cc: @LinuxBozo